Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.xaml

```diff
@@ -503,7 +518,11 @@
                     <Grid x:Name="ColumnHeaderRoot">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Normal"/>
+                                <VisualState x:Name="Normal">
+                                    <Storyboard>
+                                        <ColorAnimation Storyboard.TargetName="BackgroundRectangle" Storyboard.TargetProperty="(Shape.Fill).(SolidColorBrush.Color)" Duration="0" To="{ThemeResource DataGridColumnHeaderBackgroundBrush}"/>
```

Comment:
Daniel, this looks fishy to me:
    To="{ThemeResource DataGridColumnHeaderBackgroundBrush}"
You're animating a Color to a Brush.  All other ColorAnimation instances in this file are animating to a Color and not a Brush. The types are not compatible.
Surprisingly I don't see it causing any trouble when testing it though. What d'you think?

Reply:
You are right. PR Updated solving this issue.